spec.md

```diff
@@ -489,6 +489,34 @@ Location: https://example.com/space/81246131-69a4-45ab-9bff-9c946b59cf2e/message
 
 ### Read Resource Operation
 
+* TODO: Add language on content negotiation
+
+#### (HTTP API) GET `/spaces/{space_id}/{collection/*}{resource_name}`
+
+* Requires appropriate authorization
+    - Invoking this method via [ZCAP](#zcap) requires a capability allowing the
```

Comment:
1) I don't think a ZCAP should be required if the request is signed by the resource/space controller (flexible on this)
2) this description of the required zcap should probably have more requirements, e.g. that it is rooted in a delegation by the resource controller

Reply:
K, will clarify.

Reply:
@gobengo updated, let me know if that addresses.